sdk/cosmos/tests/setup.rs

```diff
@@ -12,9 +12,19 @@ pub fn initialize() -> Result<CosmosClient, azure_cosmos::Error> {
     Ok(client)
 }
 
+fn get_account() -> String {
+    std::env::var("COSMOS_ACCOUNT").expect("Set env variable COSMOS_ACCOUNT first!")
+}
+
 #[cfg(feature = "mock_transport_framework")]
 pub fn initialize(
     transaction_name: impl Into<String>,
 ) -> Result<CosmosClient, azure_cosmos::Error> {
-    Ok(CosmosClient::new_with_transaction(transaction_name))
+    let account_name = (std::env::var("TESTING_MODE").as_deref() == Ok("RECORD"))
```

Comment:
Are we documenting what these env vars are? At the very least it might be worth lifting all of the constants we're using in this file to the top of the file.

Reply:
They are documented [here](https://github.com/Azure/azure-sdk-for-rust/blob/main/docs/mock_transport.md), but I agree it would be nice if they all lived in constants somewhere.